ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -4,6 +4,15 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+# TODO remove unnecessary variables
+cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
+export AWS_REGION=us-east-1
+export AWS_SHARED_CREDENTIALS_FILE=$cluster_profile/.awscred
+export AZURE_AUTH_LOCATION=$cluster_profile/osServicePrincipal.json
+export AZURE_REGION=centralus
```

Comment:
Likewise, you should need the creds, but should not need to pass the region out-of-band for Azure.